
lower_threshold: [43, 70, 6]
upper_threshold: [82, 255, 255]
kernel_val: 5

# y = 0.2389192 + (4512335 - 0.2389192)/(1 + (x/127.198)^22.29046)
